src/controllers/playback/video/index.js

```diff
@@ -716,9 +716,8 @@ import { appRouter } from '../../../components/appRouter';
                     } else {
                         nowPlayingPositionSlider.value = 0;
                     }
-
-                    if (runtimeTicks && positionTicks != null && currentRuntimeTicks && !enableProgressByTimeOfDay && currentItem.RunTimeTicks && currentItem.Type !== 'Recording') {
-                        endsAtText.innerHTML = '&nbsp;&nbsp;&nbsp;&nbsp;' + mediaInfo.getEndsAtFromPosition(runtimeTicks, positionTicks, true);
+                    if (runtimeTicks && positionTicks != null && currentRuntimeTicks && !enableProgressByTimeOfDay && currentItem.RunTimeTicks && currentItem.Type !== 'Recording' && playbackRate !== null) {
+                        endsAtText.innerHTML = '&nbsp;&nbsp;&nbsp;&nbsp;' + mediaInfo.getEndsAtFromPosition(runtimeTicks, positionTicks, playbackRate, true);
```

Comment:
_A rebellious line_ :sweat_smile: 

Team wants this line in place without trailing spaces - it separates the if-blocks logically and, in theory, will simplify GH diff.
In last commit, GH decided to remove it instead of just a trailing spaces. I hope the next change lands correctly.

_You are free to squash commits about this line (this suggestion and previous) with a merge commit._
```suggestion

                    if (runtimeTicks && positionTicks != null && currentRuntimeTicks && !enableProgressByTimeOfDay && currentItem.RunTimeTicks && currentItem.Type !== 'Recording' && playbackRate !== null) {
                        endsAtText.innerHTML = '&nbsp;&nbsp;&nbsp;&nbsp;' + mediaInfo.getEndsAtFromPosition(runtimeTicks, positionTicks, playbackRate, true);
```

Reply:
The thing is whenever I add a newline, one of the checks is getting failed. That's why I removed that line in the first place.

Reply:
That check fails on trailing spaces. Some IDEs put indent automatically.

Reply:
Yes, definitely that can be a reason. The one you suggested have no trailing spaces, right?

Reply:
Yes, it is just empty line.